### ISSUE TITLE
Make search icon a button

### ISSUE THREAD

**Issue by gs0510** [ocaml/v2.ocaml.org#1325]
Currently the search icon in the header is not clickable, which can be deceiving if you go to click on it. Changing the icon to a button that can be clicked will solve this.

**Comment by 0xull** [comment]
Before now, while working on the header I discovered the icon is a background-image that was set to non-repeat, so it's not an icon at all. And putting an icon to the search component would very beneficial, countering the deceptive status.

**Comment by 0xull** [comment]
So I propose we remove the background-image and replace it with an actual SVG icon, lightweight and highly compatible with all browsers, including IE. @patricoferris @gs0510

**Comment by SanduniK95** [comment]
@patricoferris  @gs0510  Hi! I'm an Outreachy applicant and would like to work on this issue

**Comment by gs0510** [comment]
@SanduniK95 I like @IkehAkinyemi 's idea of replace the background image with an SVG icon and making it clickable. Since @IkehAkinyemi  already has worked on a `good-first-issue` you can do this. Thanks!

**Comment by pitag-ha** [comment]
@SanduniK95 , how is it going? :) Are you still working on this issue?

**Comment by SanduniK95** [comment]
@pitag-ha Yes, I'm working on it! I was having some issues with setting up the project but I'm working on it by referring to the replies of other people who seemed to have had the same problems

**Comment by guptadiksha307** [comment]
@pitag-ha Is this issue still open? If yes, I would like to work on this :)

**Comment by pitag-ha** [comment]
@guptadiksha307, I think it is still open. But notice that each Outreachy applicant can work on maximum one good-first-issue :)

**Comment by vishalsingh2972** [comment]
@gs0510 can you assign this to me if it's still open?
